src/app/clients/add/formComponents.tsx

```diff
@@ -0,0 +1,534 @@
+                                ["<1", "0"],
+                                ["1", "1"],
+                                ["2", "2"],
+                                ["3", "3"],
+                                ["4", "4"],
+                                ["5", "5"],
+                                ["6", "6"],
+                                ["7", "7"],
+                                ["8", "8"],
+                                ["9", "9"],
+                                ["10", "10"],
+                                ["11", "11"],
+                                ["12", "12"],
+                                ["13", "13"],
+                                ["14", "14"],
+                                ["15", "15"],
+                                ["Don't Know", "don't know"],
+                            ]}
+                            listTitle="Age"
+                            defaultValue="don't know"
+                            onChange={getChild(fieldSetter, fields.children, index, "age")}
+                        />
+                    </StyledCard>
+                );
+            })}
+        </>,
+        "Please enter a number between 0 and 20. (Note that children are under 16 years old)"
+    );
+
+    const dietaryRequirementCard = genericFormCard(
+        "Dietary Requirements",
+        false,
+        <CheckboxGroupInput
+            labelsAndKeys={[
+                ["Gluten Free", "Gluten Free"],
+                ["Dairy Free", "Dairy Free"],
+                ["Vegetarian", "Vegetarian"],
+                ["Vegan", "Vegan"],
+                ["Pescatarian", "Pescatarian"],
+                ["Halal", "Halal"],
+                ["Diabetic", "Diabetic"],
+                ["Nut Allergy", "Nut Allergy"],
+                ["Seafood Allergy", "Seafood Allergy"],
+                ["No Bread", "No Bread"],
+                ["No Pasta", "No Pasta"],
+                ["No Rice", "No Rice"],
+                ["No Pork", "No Pork"],
+                ["No Beef", "No Beef"],
+            ]}
+            onChange={onChangeCheckbox(
+                fieldSetter,
+                fields.dietaryRequirements,
+                "dietaryRequirements"
+            )}
+        />,
+        "Tick all that apply"
+    );
+
+    const feminineProductCard = genericFormCard(
+        "Feminine Products",
+        false,
+        <CheckboxGroupInput
+            labelsAndKeys={[
+                ["Tampons", "Tampons"],
+                ["Pads", "Pads"],
+                ["Incontinence Pads", "Incontinence Pads"],
+            ]}
+            onChange={onChangeCheckbox(fieldSetter, fields.feminineProducts, "feminineProducts")}
+        />
+    );
+
+    const babyProductCard = genericFormCard(
+        "Baby Products",
+        true,
+        <>
+            <RadioGroupInput
+                labelsAndValues={[
+                    ["Yes", "Yes"],
+                    ["No", "No"],
+                    ["Don't Know", "don't know"],
```

Comment:
Why is the value lowercased here and nowhere else?

Reply:
It's also lowercased in `children`. 
It was originally to distinguish between the label "Don't Know" and the value "don't know" but thinking back, that might not be necessary.